plugins/destination/azblob/client/spec.go

```diff
@@ -0,0 +1,39 @@
+package client
+
+import (
+	"fmt"
+)
+
+type FormatType string
+
+const (
+	FormatTypeCSV  = "csv"
+	FormatTypeJSON = "json"
+)
+
+type Spec struct {
+	Bucket   string     `json:"bucket,omitempty"`
```

Comment:
I don't think `bucket` is the correct Azure Blob Storage terminology; should probably be `storage_account`